### PR TITLE
[RFC] Updated continuous PC

### PR DESCRIPTION
**Pull request by MLasserre**
Merged the dev master to the main.
Bring the last version of continuous PC.

More documentation will be added soon

### CODE REVIEW

**Comment by MLasserre** [comment]
Sorry for opening and closing my PR.
I guess it's not how it should be done but I am not a git expert.

**Comment by jschueller** [comment]
you can stay in the same branch, just fetch upstream, rebase then force push.

File: python/src/ContinuousPC_doc.i.in [hunk 1]
```diff
@@ -1,59 +1,100 @@
 %feature("docstring") OTAGRUM::ContinuousPC
-"ContinuousPC learner.
+"ContinuousPC learner
```
Comment (jschueller) [inline python/src/ContinuousPC_doc.i.in]: @MLasserre you can keep the dots at the end, see https://www.python.org/dev/peps/pep-0257/:
`The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".`

Comment (MLasserre) [reply python/src/ContinuousPC_doc.i.in]: OK, I will fix that for when I push more doc.

Comment (jschueller) [reply python/src/ContinuousPC_doc.i.in]: so should I wait before merging ?

Comment (MLasserre) [reply python/src/ContinuousPC_doc.i.in]: Yes,  if you don't mind.

File: python/src/ContinuousBayesianNetwork_doc.i.in [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
 >>> ndag = otagrum.NamedDAG(bn)
 >>> jointDistributions = []
->>> #distribution = otagrum.ContinuousBayesianNetwork(ndag, jointDistributions)"
+>>> for i in range(ndag.getSize()):
+>>>     dim = 1 + ndag.getParents(i).getSize()
+>>>     R = ot.CorrelationMatrix(dim)
+>>>     for j in range(dim):
+>>>         for k in range(j):
+>>>             R[j, k] = 0.8
+>>>     jointDistributions.append(ot.Normal([0.0]*dim, [1.0]*dim, R).getCopula())
```
Comment (jschueller) [inline python/src/ContinuousBayesianNetwork_doc.i.in]: you need dots here:
```
>>> for i in range(ndag.getSize()):
...     dim = 1 + ndag.getParents(i).getSize()
...     R = ot.CorrelationMatrix(dim)
...
```
then it succeeds! should we merge it once you fix it ?

**Comment by MLasserre** [comment]
I have fixed the syntax in the doc and added a notebook.
We can merge now if it's ok for you.

**Comment by jschueller** [comment]
every notebook you put in python/doc/examples must be listed in the examples.rst: they are included in the sphinx documentation
also the notebooks are tested automatically, and it seems the new notebook does not run, where is compute_mean_LL ?

**Comment by MLasserre** [comment]
Now it's OK !

**Comment by jschueller** [comment]
great!

**Pull request merged**